.azure-pipelines/ultimate-pipeline.yml

```
@@ -842,21 +872,35 @@ stages:
             command: "CheckBuildLogsForErrors"
 
 - stage: exploration_tests_windows
-  condition: and(succeeded(), eq(variables['isScheduledBuild'], 'False'), eq(variables['runExplorationTests'], 'True'))
-  dependsOn: build_windows
+  condition: and(succeeded(), eq(variables['isScheduledBuild'], 'False'), or(eq(stageDependencies.check_changes.outputs['check_changes_job.check_changes_step.IsProfilerChanged'],'True'), eq(stageDependencies.check_changes.outputs['check_changes_job.check_changes_step.IsDebuggerChanged'], 'True')))
```

nit: I think these are easier to read multi-line
```suggestion
  condition: >
    and(
      succeeded(), 
      eq(variables['isScheduledBuild'], 'False'), 
      or(
        eq(stageDependencies.check_changes.outputs['check_changes_job.check_changes_step.IsProfilerChanged'],'True'), 
        eq(stageDependencies.check_changes.outputs['check_changes_job.check_changes_step.IsDebuggerChanged'], 'True')
      )
    )
```